bundles/org.openhab.automation.jsscripting/suppressions.properties

```diff
@@ -1,2 +1,3 @@
 # Please check here how to add suppressions https://maven.apache.org/plugins/maven-pmd-plugin/examples/violation-exclusions.html
 org.openhab.automation.jsscripting.internal.scriptengine.InvocationInterceptingScriptEngineWithInvocableAndAutoCloseable=AvoidThrowingNullPointerException,AvoidCatchingNPE
+org.openhab.automation.jsscripting.internal.OpenhabGraalJSScriptEngine=AvoidThrowingRawExceptionTypes
```

Comment:
Why does it need to be suppressed? Wouldn't it be possible to inherit from `RuntimeException` and throw a dedicated exception?

Reply:
Done.